README.md

```diff
@@ -223,6 +218,14 @@ This may be better suited for you if you plan to change some of the automaticall
 
    This script uses the WITHDRAW_NAME .env variable to automatically generate the selector in the necesary format
 
+
+### Note on Starknet Smart Contract
+
+If you want to use the Herodotus version of the smart contract, rename the `Escrow_herodotus.cairo` into `Escrow.cairo`. Then you must also set the following .env variable before using any deployment script:
+```env
+   HERODOTUS_FACTS_REGISTRY = Herodotus' Facts Registry Smart Contract in Starknet
+```
```

Comment:
What do you think of leaving a message that this is just a temporary solution? It's WIP and we add an issue to improve this.

Reply:
Also I would add keep the Herodotus Fact Registry constants @uri-99 added in the last PR